packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -176,9 +176,10 @@ async function initProject(appName: string, options: CLIOptions) {
         code('  create-contentful-app my-app --source "github:user/repo"'),
         '',
         `Official Contentful templates and examples are hosted at ${highlight(EXAMPLES_REPO_URL)}.`,
-        'Use',
+        'Once your app is created, use ',
```

Comment:
might be enough to add this to the readme as this is not part of the actual command.